data/CBID2.json

```
@@ -21,6 +21,11 @@
       "icon": "SiHashnode"
     }
   ],
+  {
+  "name": "Follow me on Dev.to",
+  "url": "https://dev.to/cbid2",
+  "icon": "FaFev"
+  },
```

```suggestion
    {
    "name": "Follow me on Dev.to",
    "url": "https://dev.to/cbid2",
    "icon": "FaFev"
    }
  ],
```

As a committable suggestion